.github/workflows/release-evidence-verify.yml

```
@@ -0,0 +1,134 @@
+
+          PERSISTENCE_FLAG=""
+          if [ "${{ github.event.inputs.require_persistence }}" == "true" ]; then
+            PERSISTENCE_FLAG="--require-persistence"
+          fi
+
+          LABEL_FLAG=""
+          if [ -n "${{ github.event.inputs.base_url_label }}" ]; then
+            LABEL_FLAG="--base-url-label ${{ github.event.inputs.base_url_label }}"
+          fi
+
+python scripts/check_hosted_readiness.py "$URL" --json $LABEL_FLAG $PERSISTENCE_FLAG > readiness-output.json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)